### PR TITLE
Implement `VmConfidentialClient` class

### PR DESCRIPTION
**Pull request by nesitor**
Problem: A user cannot initialize an already created confidential VM.

Solution: Implement `VmConfidentialClient` class to be able to initialize and interact with confidential VMs.

### CODE REVIEW

File: src/aleph/sdk/client/vm_client.py
Comment (1yam) [inline src/aleph/sdk/client/vm_client.py:1]: Maybe would also need rebase we made some change on it

**Review by 1yam — COMMENTED** [review]
else looks good to me. @Psycojoker what do you thinks ?

File: src/aleph/sdk/client/vm_client.py
Comment (nesitor) [reply src/aleph/sdk/client/vm_client.py:1]: I will merge that changes.

File: src/aleph/sdk/client/vm_confidential_client.py [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+            self.pubkey_signature_header = (
+                await self._generate_pubkey_signature_header()
+            )
+
+        operation = "confidential/measurement"
+        url, header = await self._generate_header(vm_id=vm_id, operation=operation)
+        try:
+            async with self.session.get(url, headers=header) as response:
+                response = await response.json()
+                print(response)
```
Comment (1yam) [inline src/aleph/sdk/client/vm_confidential_client.py:110]:
```suggestion
```

Comment (1yam) [reply src/aleph/sdk/client/vm_confidential_client.py:110]: Look like a debug print

**Review by 1yam — APPROVED** [review]
Else LGTM

**Review by olethanh — APPROVED** [review]

**Pull request merged**